docassemble/ALKilnInThePlayground/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.1.1'
+__version__ = '1.2.1'
```

Comment:
Should this be `1.2.0`? I'm assuming there hasn't been one yet, since it'd have to come from here.

Reply:
It should be, thanks!

```suggestion
__version__ = '1.2.0'
```